tests/compute/vmidefaults.go

```diff
@@ -243,8 +245,7 @@ var _ = SIGDescribe("VMIDefaults", func() {
 			Expect(err).ToNot(HaveOccurred())
 
 			By("Getting VirtualMachineInstance")
-			vmi, err = virtClient.VirtualMachineInstance(testsuite.GetTestNamespace(vm)).Get(context.Background(), vm.Name, metav1.GetOptions{})
```

Comment:
Doesn't the `Get` needs to stay for latter asserts

Reply:
Need fresh air :man_facepalming: 
Thank you